common/flatpak-installation.c

```diff
@@ -1024,76 +1052,114 @@ flatpak_installation_list_installed_refs_for_update (FlatpakInstallation *self,
                                                      GCancellable        *cancellable,
                                                      GError             **error)
 {
-  g_autoptr(FlatpakDir) dir_orig = flatpak_installation_get_dir_maybe_no_repo (self);
-  g_autoptr(FlatpakDir) dir = NULL;
-  g_auto(GStrv) remote_names = NULL;
-  g_autoptr(GPtrArray) updates = NULL; /* (element-type FlatpakInstalledRef) */
-  g_autoptr(GHashTable) remote_states = NULL; /* (element-type utf8 FlatpakRemoteState) */
-  g_auto(GStrv) refs_app = NULL;
-  g_auto(GStrv) refs_runtime = NULL;
+  GLNX_AUTO_PREFIX_ERROR (G_STRFUNC, error);
```

Comment:
Is it really useful to expose function names in error messages which are meant for human display?

Reply:
I think it makes the error messages more useful when the goal is to debug the error.

Reply:
Thats not false, but typically GErrors are shoved into dialogs shown to users, where it is less than ideal.

Reply:
Deleted this line